blog/_posts/2016-08-20-hello-from-team-Crackers.md

```diff
@@ -0,0 +1,53 @@
+---
+title: hello from Team Crackers!!
+layout: post
+created_at: Sat August 20 2016
+permalink: blog/2016-08-20-hello-from-team-Crackers
+current: blog
+author: Nikita and Nishtha
+twitter: Team Crackers
+categories:
+- student-posts
+---
+
+# **Team Crackers**
+
+![Team Crackers](/img/blog/2016/teamCrackers1.jpg)    
+
```

Comment:
please add image credits as explained in the [readme](https://github.com/rails-girls-summer-of-code/summer-of-code#images) to all your photos/images.